tests/callbacks/test_mlperf_callback.py

```diff
@@ -0,0 +1,65 @@
+import logging
+
+import numpy as np
+import pytest
+from torch.utils.data import DataLoader
+
+from composer import Trainer
+from composer.callbacks import MLPerfCallback
+from tests.common import RandomClassificationDataset, SimpleModel
+
+logging.basicConfig(filename="/Users/hanlintang/composer/package_checker.log", level=logging.INFO)
+logging.getLogger().addHandler(logging.StreamHandler())
+formatter = logging.Formatter("%(levelname)s - %(message)s")
+logging.getLogger().handlers[0].setFormatter(formatter)
+logging.getLogger().handlers[1].setFormatter(formatter)
+
```

Comment:
Can we add in a test to ensure the right things are logged? Thinking that we could monkeypatch-and-capture the `_log_dict` method to ensure that the expected calls are made (or do something similar there).

Reply:
Is it sufficient that we used the `package_checker` below from MLPerf, which is what validates that the log is compliant?